src/test/java/com.epam.tc.api.hw2/LowTests.java

```diff
@@ -0,0 +1,180 @@
+package com.epam.tc.api.hw2;
+
+import static org.hamcrest.Matchers.equalTo;
+
+import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import org.testng.annotations.Test;
+
+
+
+public class LowTests extends TestBase {
+
+    public static String boardId;
+    public static String listId;
+    public static String cardId;
+
+    @Test(priority = 1)
```

Comment:
It is better to avoid dependent tests. If one of them fails for some reason, e.g. board is not created due to connection issues, all other tests won't pass. Also it's more difficult to maintain them when the amount of tests is quite big. Another reason - sometimes you need to run tests separately, e.g. run tests related to Lists behavior without Board tests. So, this class should be reworked to make tests independent.